src/chapters/auth/scaffolding.rst

```diff
@@ -38,45 +38,43 @@ When you run this command, the output may look something like the following:
 .. sourcecode:: guess
```

Comment:
Instead of an extra tab, how about we revisit the styling later to see if we can increase the space? That way we can make sure that everything is consistently spaced out